Formula/ceph-client.rb

```diff
@@ -0,0 +1,103 @@
+class CephClient < Formula
+  desc "Rados and RBD CLIs and libraries of the Ceph project"
+  homepage "https://ceph.com"
+  url "git@github.com:ceph/ceph.git", :using => :git, :tag => "v13.2.1", :revision => "5533ecdc0fda920179d7ad84e0aa65a127b20d77"
```

Comment:
@zeichenanonym how about using
```
url "https://github.com/ceph/ceph.git", :tag => “v13.2.1”,
                                        :revision => "5533ecdc0fda920179d7ad84e0aa65a127b20d77"
```

instead of using `ssh`?

Reply:
The tarball lacks the git submodules - that's why it was easier for me to pull it from git... using https might solve the tests - will push an update to the branch using https tonight

Reply:
how about `
https://download.ceph.com/tarballs/ceph_13.2.1.orig.tar.gz` ?